data/hai/hai.txt

```diff
@@ -702,3 +717,194 @@ phrase "hostile disabled unfettered"
+		"Remember, you only need to give me a symbolic fee for this to stop."
+		"This is not a duel to the death, or else only one of us would be able learn."
+		"Do you wish to surrender?"
+		"I am reasonable, if you wish to surrender, we can come to terms."
+		"You are lucky that I have to accept any ceasefire if you ask... You would do well not to attempt anything."
+
+phrase "defeated unfettered challenger"
+	word
+		"You have "
+		"You succeeded in "
+		"You managed to have "
+		"Somehow, you have "
+		"I must admit you have "
+	word
+		"beaten "
+		"defeated "
+		"ashamed "
+		"disabled "
+		"revealed my weakness% "
+	word
+		"me in a% "
+	word
+		"glorious "
+		"pathethic "
+		"exceptional "
+		"grandiose "
+		"quick "
+		"strategic "
+	word
+		"battle"
+		"combat"
+		"duel"
+		"fight"
+		"challenge"
+	word
+		"."
+		"!"
+	word
+		" What more do you want from me?"
+		" Do not presume you may board me."
+		" Do not even think about capturing my ship."
+		" Now it is over, time to lay down your weapons."
+		" I accept my fate, take what is rightfully yours."
+		" Great stories will be told of our battle!"
+		" You have the right to plunder."
+		" It would seem you are as strong as some pretended, perhaps even more."
+		" Finish me! I would rather die than have to live with this insult."
+	replace
+		"% me" ""
+		"a% e" "an e"
+		"a%" "a"
+
+phrase "unfettered duelling"
+	word
+		"Do not presume to interrupt us"
+		"I am in the middle of a duel, there is no time for petty talk"
+		"This is an official challenge, do not intervene"
+		"Under no circumstances are you to interrupt us, or you will suffer the wratch of the Unfettered"
+		"Stop distracting me"
+		"If you try anything the whole fleet will be on you"
+	word
+		"! "
+	word
+		"Go away now, I need to focus on the duel"
+	word
+		"!"
+		"."
+
+phrase "friendly mercenary"
+	word
+		"What is your command"
+		"Orders"
+		"Anything I can do for you"
+		"Who should I shoot"
+		"Covering fire, where"
+		"Yes"
+		"Change of plan"
+		"I am ready"
+	word
+		", "
+	word
+		"Boss"
+		"Leader"
+		"Alpha"
+		"First Mercenary"
+		"Commander"
+		"Captain"
+	word
+		"?"
+
+phrase "disabled mercenary"
+	word
+		"I await judgment."
+		"Do not expect me to beg."
+		"It seems I was proven... weak."
+		"I can pay you... If you would only lean me a hand."
+		"Truly a shameful display on my part."
+		"I have executed the former captain for his incompetence."
+		"Leave me alone, I do not deserve any help."
+
+phrase "enemy mercenary"
+	word
+		"You have "
+		"You possess "
+		"On your ship is "
+	word
+		"something "
+		"an item "
+		"a possession "
+	word
+		"precious to us"
+		"valuable to us"
+		"of mine "
+		"that we want "
+	word
+		"."
+		"!"
+	word
+		" Surrender it"
+		" Drop it in the next spaceport"
+		" Give it to me and your death will be quick."
+		" Nemo Me Impune Lacessit"
+		" Sell it on a lawless world"
+		" Give it back, scum"
```

Comment:
`Give it back` only works with `of mine`; maybe it should be `Give it to us` or `Hand it over`?